### ISSUE TITLE
Hinge Loss for generator needs to be fixed

### ISSUE THREAD

**Issue by EmanueleGhelfi** [zurutech/ashpy#23]
Line: https://github.com/zurutech/ashpy/blob/f05353285a80d88ad24a41344d361040b356f464/ashpy/keras/losses.py#L266

The loss is calculated as - tf.nn.relu(d_fake)

However, as in SA-GAN:
https://github.com/brain-research/self-attention-gan/blob/ad9612e60f6ba2b5ad3d3340ebae60f724636d75/model.py#L73

we should only minimize - d_fake

**Comment by galeone** [comment]
Closed by https://github.com/zurutech/ashpy/pull/24
